### PR TITLE
RW-Update assign_service_templates.md

### PR DESCRIPTION
**Pull request by rwellon**

### CODE REVIEW

**Comment by rwellon** [comment]
@miq-bot add-label najdorf/yes?

File: api/examples/assign_service_templates.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 #### Request:
 
-    POST /api/service_catalogs/6/service_templates
+    POST /api/service_catalogs/1/service_templates
```
Comment (Fryguy) [inline api/examples/assign_service_templates.md:8]: I just noticed, but I think this request is wrong and should be 

```suggestion
    POST /api/service_catalogs/6/service_catalogs
```

**Comment by Fryguy** [comment]
@rwellon What problem is trying to be solved by changing the id number from `6` to `1`?  Can we leave the id number?

**Comment by rwellon** [comment]
@Fryguy This template being labelled as `6` when being referenced as Service Catalog 1 caused confusion

**Comment by Fryguy** [comment]
Backported to `najdorf` in commit aaafc87f7480060eec81d22a3af20d77fb11c4f2.

```text
commit aaafc87f7480060eec81d22a3af20d77fb11c4f2
Author: Jason Frey <fryguy9@gmail.com>
Date:   Thu May 5 13:47:53 2022 -0400

    Merge pull request #1663 from rwellon/RW-UpdateAPI-ExampleTemplate
    
    RW-Update assign_service_templates.md
    
    (cherry picked from commit 77e8bfab621942313559d23c46f55f1005687cd7)
```

**Pull request merged**